SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -434,10 +434,9 @@ FRepChangeState USpatialActorChannel::CreateInitialRepChangeState(TWeakObjectPtr
 		const FRepLayoutCmd& Cmd = Replicator.RepLayout->Cmds[CmdIdx];
 		const FRepChangedParent& Parent = Replicator.RepState->GetSendingRepState()->RepChangedPropertyTracker->Parents[Cmd.ParentIndex];
 
-		// HACK: Need special case here because the gdk relies on the whole of the RepMovement struct being replicated to the client to decide whether to replicate physics.
-		// see: ComponentReader::ApplySchemaObject
-		// UNR-5843 TODO: fix this so we no longer need this special handling, for instance by replicating bRepPhysics as a separate always
-		// replicated field.
+		// HACK: Need special case here because the gdk relies on the whole of the RepMovement struct being replicated to the client to
+		// decide whether to replicate physics. see: ComponentReader::ApplySchemaObject UNR-5843 TODO: fix this so we no longer need this
+		// special handling, for instance by replicating bRepPhysics as a separate always replicated field.
```

Comment:
```suggestion
		// HACK: Need special case here because the gdk relies on the whole of the RepMovement struct being replicated to the client to decide whether to replicate physics.
		// see: ComponentReader::ApplySchemaObject

		// UNR-5843 TODO: fix this so we no longer need this special handling, for instance by replicating bRepPhysics as a separate always
		// replicated field.
```